From daf96b07c02734dceaf3c3b78c10e4e758b74d9e Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@e-labworks.com>
Date: Fri, 18 Dec 2015 15:08:58 -0200
Subject: [PATCH 1/1] Fix compile error when cross-compiling.

When cross-compiling with libjpeg enabled, we can get errors like the
following when linking:

sh-linux-gnu-gcc: ERROR: unsafe header/library path used in
cross-compilation: '/lib'

That's because there is an error in the configure script that are not
generating ldflags correctly, and are trying to link with -L/lib.

Signed-off-by: Sergio Prado <sergio.prado@e-labworks.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 772d5fc9d87a..58d55ade48e3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -223,7 +223,7 @@ AC_ARG_WITH([libjpeg],
         jpeg_ok=no)
       AC_MSG_RESULT($jpeg_ok)
       if test "$jpeg_ok" = yes; then
-        JPEG='jpeg'; LIBJPEG='-L${libjpeg_prefix}/lib -ljpeg'
+        JPEG='jpeg'; LIBJPEG="-L${libjpeg_prefix}/lib -ljpeg"
       else
         AC_MSG_WARN(*** JPEG loader will not be built (JPEG header file not found) ***)
       fi
--
1.9.1

